server/runtime/transcript_runtime_state.go

```
@@ -53,6 +53,15 @@ func (s *transcriptRuntimeState) chatProjection() *chatStore {
 	return s.chat
 }
 
+func (s *transcriptRuntimeState) replaceChatProjection(chat *chatStore) {
```

golangci-lint (2.12.2)

[error] func (*transcriptRuntimeState).replaceChatProjection is unused

(unused)